src/applications/coronavirus-chatbot/utils.js

```diff
@@ -7,6 +15,8 @@ const disableButtons = event => {
   for (let i = 0; i < siblingButtons.length; i++) {
     siblingButtons[i].disabled = true;
   }
+
+  scrollToNewMessage();
```

Comment:
I'd recommend either
1. Adding a separate event listener for this after line 26, or
2. Creating a new `questionAnswered` function which calls both `disableButtons` and `scrollToNewMessage`

Reply:
We're implementing option 2!